shopping-cart/shopping-cart-java/shopping-cart/src/test/java/com/example/shoppingcart/impl/ShoppingCartReportTest.java

```diff
@@ -56,8 +56,6 @@ public static void afterAll() {
     public void createAReportOnFirstEvent() throws InterruptedException, ExecutionException, TimeoutException {
 
         String cartId = UUID.randomUUID().toString();
-        assertNull(Await.result(reportRepository.findById(cartId)));
```

Comment:
This kind of assertions was causing the build to eventually fail. When we reach that line, it's possible that the read-side processor didn't get a chance to create the user tables and then it will fail on executing this selection. 

I'm removing it because it's not strictly necessary for the tests. However, we need to rethink the test driver. Users can easily run into that situation.

Reply:
I opened an issue to address this.
https://github.com/lagom/lagom/issues/2248